ee/candi/cloud-providers/openstack/layouts/standard/base-infrastructure/main.tf

```diff
@@ -62,7 +62,7 @@ locals {
   zone               = lookup(local.bastion_instance, "zone", null) != null ? local.bastion_instance.zone : local.actual_zones[0]
   metadata_tags      = merge(lookup(var.providerClusterConfiguration, "tags", {}), lookup(local.instance_class, "additionalTags", {}))
   config_drive       = false
-  root_disk_size     = lookup(local.instance_class, "rootDiskSize", "")
+  root_disk_size     = lookup(local.instance_class, "rootDiskSize", "50")
```

Comment:
i think we can decrease bastion size to 30 gb or less (20, but i think it will be not actual because OS image can have ~20 gb, 30 is optimal), because it does not contain any images, only OS